src/style/ToDoList.scss

```diff
@@ -0,0 +1,107 @@
+@import "../index.scss";
+
+// Main Styling for all the ToDo's
+.tasklist {
+  padding: 0 30px;
+
+  ul {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+
+    li {
+      list-style: none;
+      padding: 10px;
+      border-radius: 10px;
+      margin-bottom: 10px;
+      display: flex;
+      gap: 5px;
+      width: 300px;
+      align-items: center;
+
+      input {
+        width: 25px;
+        height: 25px;
+        cursor: pointer;
+      }
+
+      .single-task {
+        font-size: 15px;
+      }
+
+      .fireball {
+        width: 20px;
+        height: 20px;
+      }
+
+      .date {
+        font-size: 12px;
+        margin-left: auto;
+        width: 50px;
+      }
+
+      button {
+        margin-left: auto;
+        font-size: 20px;
+        border: none;
+        background: transparent;
+        cursor: pointer;
+      }
+    }
+  }
+}
+
+// Extra Light Mode Styling
+.background-light {
+  border: 10px solid $light-sec;
+  background-color: $light-fourth;
+}
+
+// Extra Dark Mode Styling
+.background-dark {
+  border: 10px solid $dark-fourth;
+  background-color: $dark-prim;
+  color: #fff;
+}
+
+// Responsive Styling
+@media (min-width: 460px) {
```

Comment:
We are wondering why you chose 460px and 820px as the breakpoints. Could you kindly explain it?

//Izabel & Wen